source/extensions/filters/http/aws_lambda/aws_authenticator.cc

```diff
@@ -60,6 +60,11 @@ void AwsAuthenticator::updatePayloadHash(const Buffer::Instance &data) {
   body_sha_.update(data);
 }
 
+void AwsAuthenticator::resetBodySha() { 
```

Comment:
this seems to be unused? remove?

Reply:
Removed this method here: https://github.com/solo-io/envoy-gloo/pull/177/commits/70e326a0592159c972822643eb51ab8425fbf836